medicines/doc-index-updater/src/create_manager/sftp_client.rs

```diff
@@ -87,6 +87,23 @@ async fn retrieve_file_from_sftp(
         Err(_) => path,
     };
     tracing::info!("{:?}", path);
+
+    // Additional logging to debug observed sftp issue
+    // in nonprod environment
+    let parent_dir = path.parent();
+    if let Some(parnet_dir_path) = parent_dir {
+        tracing::debug!("Finding contents of {:?}", &parnet_dir_path);
+        if let Ok(file_stats) = sftp.readdir(parnet_dir_path) {
+            for (path_buf, file_stat) in file_stats {
+                tracing::debug!("{:?}", path_buf.to_str());
```

Comment:
Is this outputting the file name or file contents?

Reply:
It's the file path.

Reply:
and maybe expand buf -> buffer ?    if this code is only temporary then no big deal